OsmAnd/src/net/osmand/plus/routing/RoutingHelper.java

```diff
@@ -260,6 +262,17 @@ public LatLon getFinalLocation() {
 		return finalLocation;
 	}
 
+	public void cacheStartLocation(Location nextStartLocation) {
```

Comment:
cacheStartLocation method name is wrong ?

Reply:
fixed